.github/workflows/test-server.yml

```
@@ -12,7 +12,7 @@ on:
       server_tag:
         description: The docker tag to use when pulling the server
         type: string
-        default: 3.3.1
+        default: latest
```

A comment might be good to help us remember this in the future?

```suggestion
        # Consider updating this in the calling workflow: https://github.com/realm/realm-js/blob/1a1a0edc354059122f1d9800c66a4a0860120fcb/.github/workflows/pr-realm-js.yml#L245
        default: latest
```